actpool/actpool.go

```diff
@@ -230,7 +239,11 @@ func (ap *actPool) Add(ctx context.Context, act *action.SealedEnvelope) error {
 		return err
 	}
 
-	if err := ap.checkSelpWithoutState(ctx, act); err != nil {
+	bcCtx, err := ap.chainContext(ctx)
+	if err != nil {
+		return err
+	}
+	if err := ap.checkSelpWithoutState(bcCtx, act); err != nil {
```

Comment:
any changes in `checkSelpWithoutState`?

Reply:
didn't see any place to set `gasFeeCap` of action?

Reply:
how does the 1559 affect the action sorting in actpool?

Reply:
> any changes in `checkSelpWithoutState`?

It is in the validator called in this function, but I don't think it is valid: https://github.com/iotexproject/iotex-core/pull/4353#discussion_r1704853410

Reply:
fixed in latest commit